.github/workflows/nf-test-gpu.yml

```
@@ -70,7 +70,7 @@ jobs:
           echo ${{ steps.set-shards.outputs.total_shards }}
 
   nf-test:
```

```suggestion
  nf-test-gpu:
```
makes it easier to differentiate

---

https://github.com/nf-core/sarek/pull/1851 since we forgot to update the expected `nf-test` :facepalm: